Automattic/jetpack
# Update/jitm tracking props

## Proposed changes:

* This diff moves the tracking for JITM buttons into the JITM package and adds a message_path property to the event props.

## Does this pull request change what data or activity we track or use?


Yes: this change includes additional properties on JITM view and click events, noting which page in the wp-admin the JITM was displayed or clicked on.

## Testing instructions:






* Apply this PR to your Jetpack test site
* Make sure you build the following packages: `jetpack build packages/jitm` `jetpack build packages/tracking`
* Build the Jetpack plugin: `jetpack build plugins/jetpack` 
* Navigate to the Jetpack dashboard of your test site. If you have not dismissed it already in the past, you should see a JITM message show at the top of the page about the Jetpack Mobile App (or another JITM):
![Screen Shot 2021-12-22 at 4 10 41 PM](https://user-images.githubusercontent.com/18016357/147156869-96e7391a-febd-4681-b39b-e77827eebe8d.png)

* Click on the JITM CTA button
* Wait a few minutes and confirm that Tracks has received the events for `jitm_view_client` and `nudge_click` and that both have the `jitm_message_path` prop set.
